x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.test.tsx

```diff
@@ -224,7 +224,10 @@ describe('unified data table', () => {
     SPECIAL_TEST_TIMEOUT
   );
 
-  it(
+  // SKIP: This test runs forever and never completes
+  // after https://github.com/elastic/kibana/pull/191249
+  // but the actual functionality works fine
+  it.skip(
```

Comment:
@elastic/security-threat-hunting-investigations (maybe @logeekal or @michaelolo24 specifically?) For some reason after these changes, this test consistently fails and hangs until CI times out (locally it just ran forever until I killed the process). It seems to have something to do with the `fireEvent.change` call below, but I couldn't pinpoint what was happening. As far as I can tell the actual functionality isn't impacted and Timeline works as expected, plus it's the only Jest test failing, so maybe you could confirm things work as expected and this could be investigated as a followup if all seems good?

Reply:
Thanks @davismcphee , I will pull down the branch and check what is the issue. If something needed to be fixed in the test, I will push that..

Reply:
Just to add-on, I think it might EUI data grid test mode creating issue with memoized components. I did try to memoize render cell value before when we were working on improving the performance of unified data table but I had to remove it becuase Euidatagrid was not playing nice with it in jest [as you can see here](https://github.com/elastic/kibana/pull/184747/files#diff-795b324c9b066e6ec4751980f32f51c0c92cf258e21161f73f2e7ba8db95be95).

But I will check again to confirm that it is the same issue.

Reply:
I looked into this more and found the warnings were related to incorrect prop type validation generated by EUI for Jest testing components, specifically from [this file](https://github.com/elastic/eui/blob/23bef2b8d11edf959311fc6bfe7161b51fbcd2cd/packages/eui/scripts/babel/proptypes-from-ts-props/index.js). I addressed it in this commit by skipping memoization for Jest environments, which gets rid of the warnings: https://github.com/elastic/kibana/pull/191249/commits/b6ceeb14d4636ace13a424505b70ff1d693d2008.

But unfortunately it looks like this wasn't the source of the original issue and this test still runs forever, so I'm really not sure what could be going on here. I even resolved all of the warnings in our Unified Data Table tests to be sure I wasn't missing something, but they're warning free now with no similar issues occurring.

Reply:
Thanks @davismcphee for looking into it and adding JEST clause. I should have done that initially but it did not occur to me. 

I have fixed the test here: d93e90d909118a2e8d49b9d7b6fe3aadfc308317.

Issue was huge amount of data in `jsdom`. Since changing `rowHeight` re-renders the complete table, it was causing jsdom to take huge amount of time to re-render and thus `firevent.change` never reports that the value has been changed.

I changed the passed data to just one row and test passed.

Regarding `NaN` warning, I will check it separately, it comes everytime the data grid renders, probably will need to mock some EuiDataGrid height calculation function.

Reply:
Thanks for looking into that and fixing the test @logeekal!